pyvista/utilities/reader.py

```diff
@@ -2006,6 +2008,25 @@ class SegYReader(BaseReader):
     _class_reader = staticmethod(_vtk.lazy_vtkSegYReader)
 
 
+class XdmfReader(BaseReader, PointCellDataSelection):
+    """XdmfReader for .xdmf files.
+
+    Examples
+    --------
+    >>> import pyvista
+    >>> from pyvista import examples
+    >>> filename = examples.download_dolfinx_mesh(load=False)
+    >>> reader = pyvista.get_reader(filename)
```

Comment:
There is a document build error in this section. I don't get the reason. @MatthewFlamm Do know the reason?

Reply:
I believe a custom vtk wheel is used for the documentation build.  Could this be the reason?

Reply:
```suggestion
    >>> # skip the following because of https://github.com/pyvista/pyvista/pull/2839/files#r903307641
    >>> reader = pyvista.get_reader(filename)   # doctest:+SKIP
```